src/app_engine/apprtc.py

```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
+    'apprtc.appspot.com', 'apprtc.webrtc.org', 'www.appr.tc'
+  ]
+  ARGUMENTS = self.request.arguments()
+  parsed_args = ''
+  if self.request.headers['Host'] in REDIRECT_DOMAINS:
+    webapp2.redirect('https://appr.tc', permanent=True)
```

Comment:
Why do a redirect here and then another one at line 606? It seems the one at line 606 is the correct one and that this one should be removed. Or have a I misunderstood how webapp2.redirect works? I tried finding documentation but couldn't find anything pointing at any reason using it like this.